stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/model/AttachmentMetaData.kt

```diff
@@ -20,6 +20,8 @@ public data class AttachmentMetaData(
     var isSelected: Boolean = false
     var selectedPosition: Int = 0
     var videoLength: Long = 0
+    var width: Int = 0
+    var heigth: Int = 0
 
```

Comment:
1. Changelog should be updated about this change.
2. I'd appreciate having docs explaining what state of those props user can expect when the Attachment is not an image (e.g. mp3 file)
3. Shouldn't we add @JvmOverloads to the constructor given we have bunch of default values set here?

Reply:
Makes sense 👍 . For now let's hold this PR a bit until we add support for autosizable images. This is more to show that it is possible to send the dimensions than the final solution.